core/src/stored/stored_conf.cc

```diff
@@ -186,7 +186,7 @@ static ResourceItem dev_items[] = {
   {"LabelBlockSize", CFG_TYPE_PINT32, ITEM(res_dev, label_block_size), 0, CFG_ITEM_DEFAULT,
       "64512" /* DEFAULT_BLOCK_SIZE */, NULL, NULL},
   {"MinimumBlockSize", CFG_TYPE_PINT32, ITEM(res_dev, min_block_size), 0, 0, NULL, NULL, NULL},
-  {"MaximumBlockSize", CFG_TYPE_MAXBLOCKSIZE, ITEM(res_dev, max_block_size), 0, 0, NULL, NULL, NULL},
+  {"MaximumBlockSize", CFG_TYPE_MAXBLOCKSIZE, ITEM(res_dev, max_block_size), 0, CFG_ITEM_DEFAULT, "1048576", NULL, NULL},
   {"MaximumFileSize", CFG_TYPE_SIZE64, ITEM(res_dev, max_file_size), 0, CFG_ITEM_DEFAULT, "1000000000", NULL, NULL},
```

Comment:
Maybe increase the default for Maximum File Size to something bigger for modern tape drives?

Reply:
That's probably a good idea, but right now I don't have a great default value that will work well in any case.
The culprit is stopping the tape (which could be mitigated by making the filemark non-immediate). Once that's done 10 GiB or 25 GiB is probably a good default. Until then I'm not sure - maybe 100 GiB or 250 GiB. We're trading restore seek-time for backup-speed here :(